study.md

```diff
@@ -58,7 +65,9 @@ console.log(`Many people drink ${beverage}.`);
 Write some Ruby code to duplicate this functionality.
 
 ```ruby
-# your answer here
```

Comment:
This will cause an error.  `def ... end` is used to define functions.  You did get the string interpolation part correct, though.

```ruby
beverage = 'tea'

puts "Many people drink #{beverage}."
```